Made adjustements to the composer json file and also adjusted the com…

BusaniPrepaid opened the pull request:
…ponent so that its compatible with cakephp45

BusaniPrepaid committed:
Made adjustements to the composer json file and also adjusted the component so that its compatible with cakephp45

ivanamat commented:
Thank you @BusaniPrepaid

ivanamat reviewed (commented):
Looks good to me

ivanamat commented:
Hi @BusaniPrepaid,

I changed the repository name to cakephp-captcha, updated README.md and added you as a contributor. On the other hand, I have also modified packagist, changing the package name and updating to version 2.0.

Regards!

BusaniPrepaid commented:
Awesome, thank you so much!

On Thu, Apr 4, 2024 at 5:23 PM Iván ***@***.***> wrote:

> Hi @BusaniPrepaid <https://github.com/BusaniPrepaid>,
>
> I changed the repository name to cakephp-captcha, updated README.md and
> added you as a contributor. On the other hand, I have also modified
> packagist, changing the package name and updating to version 2.0.
>
> Regards!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ivanamat/cakephp-captcha/pull/2#issuecomment-2037509968>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BFLFSAMSYOM3BAFGCGZWXD3Y3VV5TAVCNFSM6AAAAABFGU5ZR2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAMZXGUYDSOJWHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
[image: Company logo]
Busani Moyo
PHP Developer
Enbaya Prepaid Meters (Pty) Ltd
P: +27 (010) 500 1110 <+27%20%28010%29%20500%201110>
M:
E: ***@***.***
W: www.prepaidmeters.com
[image: -] [image: Facebook icon]
<https://www.facebook.com/PrepaidMeters/> [image:
Twitter icon] <https://twitter.com/prepaidmeters> [image: Linkedin icon]
<https://www.linkedin.com/company/prepaidmeters-co-za> [image: Instagram
icon] <https://www.instagram.com/prepaidmeters>

[image: TID Automation]
Legal Disclaimer: This message contains privileged and confidential
information intended only for the use of the addressee named above. If you
are not the intended recipient of this message, you are hereby notified
that you must not disseminate copy or take any action in reliance on it. If
you have received this message in error, please notify Enbaya Management
***@***.***> via email immediately. Any views expressed in
this message are those of the individual sender, except where the sender
specifically states them to be the view of Enbaya Prepaid Meters (Pty) Ltd.
While every care has been taken in preparing this document, no
representation, warranty or undertaking (express or implied) is given. No
responsibility or liability is accepted by Enbaya Prepaid Meters (Pty) Ltd
as to the accuracy of the information contained herein, or for any loss
arising from reliance on it. Copyright Enbaya Prepaid Meters (Pty) Ltd. All
rights reserved.
